### DISCUSSION TITLE
Only show costum shortcuts.

### DISCUSSION THREAD

**Discussion by MariusNielsen** [Anze/KeyCluCask#95] [Q&A]
Hi I really love the project. I am wondering if it is possible to make it so that KeyClu only shows my custom shortcuts?

I have quite a few keymaps through SKHD, for stuff like opening my email client and such. For this reason I would love to be able to blacklist certain applications or enabling only seeing my own short cuts (although this is the much less desirable solutions).

Additionally many of my shortcuts use return and space and KeyClu does not seem to parse these.

**Comment by Anze** [comment]
Hey [MariusNielsen](https://github.com/MariusNielsen),
Thanks for your message!
Regarding your first question: if you're referring to SKHD shortcuts, then yes, you can see only those. You'll need to trigger the window using a different shortcut that you’ve defined in SKHD, not the default one (please check [wiki page](https://github.com/Anze/KeyCluCask/wiki/Integrations-%E2%80%90-CLI-params) for the command). If you're talking about custom shortcuts set via the 'My Shortcuts' section in the app, that isn't possible at the moment, but it could be implemented in the future.
As for the second topic, could you please provide a sample of your config with the shortcuts you're having trouble with?

**Reply by MariusNielsen** [reply]
Hi,

That is exactly what I wanted. Although, I think my problem is I am in the situation of https://github.com/Anze/KeyCluCask/issues/94 as my SKHD shortcuts do not show up. Both in the normal KeyClu sheet and in my SKHD only.

My skhdrc is symlinked, to the suggested places. However I have added the path to the dict as suggested in additional configuration.

For the 2nd topic. I think I have fixed it.
